---
string resultFilenameSmall
string resultFilenameBig
---
string executing
uint8 commandsToGo